packages/runtime/src/widgets/Form/MultiSelect.tsx

```diff
@@ -52,13 +51,23 @@ export type MultiSelectProps = {
   hintStyle?: EnsembleWidgetStyles;
   allowCreateOptions?: boolean;
 } & EnsembleWidgetProps<MultiSelectStyles> &
-  FormInputProps<string[]>;
+  FormInputProps<object[] | string[]>;
+
+interface SelectOption {
+  label: Expression<string> | { [key: string]: unknown };
+  value: Expression<string | number>;
```

Comment:
These types aren't right? Label should be `Expression<string> | EnsembleWidget`?

value should be `Expression<string | number | object>`.